pkg/3scale/amp/template/adapters/backend.go

```diff
@@ -34,11 +36,22 @@ func (b *Backend) Objects() ([]common.KubernetesObject, error) {
 }
 
 func (b *Backend) options() (*component.BackendOptions, error) {
-	bob := component.BackendOptionsBuilder{}
-	bob.AppLabel("${APP_LABEL}")
-	bob.SystemBackendUsername("${SYSTEM_BACKEND_USERNAME}")
-	bob.SystemBackendPassword("${SYSTEM_BACKEND_PASSWORD}")
-	bob.TenantName("${TENANT_NAME}")
-	bob.WildcardDomain("${WILDCARD_DOMAIN}")
-	return bob.Build()
+	bo := component.NewBackendOptions()
+	bo.AppLabel = "${APP_LABEL}"
+	bo.SystemBackendUsername = "${SYSTEM_BACKEND_USERNAME}"
+	bo.SystemBackendPassword = "${SYSTEM_BACKEND_PASSWORD}"
+	bo.TenantName = "${TENANT_NAME}"
+	bo.WildcardDomain = "${WILDCARD_DOMAIN}"
+	bo.RouteEndpoint = fmt.Sprintf("https://backend-%s.%s", "${TENANT_NAME}", "${WILDCARD_DOMAIN}")
```

Comment:
See related review comment in the options backend options provider about making this a default function

Reply:
done